tests/specs/scss/_contains.test.scss

```diff
@@ -0,0 +1,24 @@
+@include test-module("contains") {
+  // Testing char-at functions
+  @include test("contains [function] should return true if key exist.") {
+    $expect: true;
+    $testlist: body 5, heading (blue red, green yellow);
```

Comment:
Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `yellow` should only be used in variable declarations; they should be referred to via variable everywhere else.